packages/cli/src/commands/org/resources/monitor.ts

```diff
@@ -100,8 +100,8 @@ export default class Monitor extends Command {
     const {flags} = await this.parse(Monitor);
     return {
       wait: flags.wait,
-      // TODO: revisit with a progress bar once the response contains the remaining resources to process
-      onRetryCb: (report: ResourceSnapshotsReportModel) => this.refresh(report),
+      // TODO: CDX-949: revisit with a progress bar once the response contains the remaining resources to process
```

Comment:
`onRetryCb`: Callback to execute every time a polling request is being made to retrieve the snapshot data.
Currently, it is only used by the monitor command so we can get the latest snapshot report after every request.

With a progress bar, we don't really need to read the snapshot report after every poll since the API returns the snapshot progress